modules/cdk-app/lib/cdk-demo-stack.ts

```diff
@@ -0,0 +1,15 @@
+import cdk = require('@aws-cdk/core');
+import s3 = require('@aws-cdk/aws-s3');
+
+export class DemoCdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string) {
+    super(scope, id);
+
+    new s3.Bucket(this, 'reInventDemoBucket', {
+      versioned: true,
+    });
+  }
+}
+
+const app = new cdk.App();
+new DemoCdkStack(app, "DemoCdkStack");
```

Comment:
It looks like `cdk-demo.ts` imports this file and does these same 2 lines of code... So you're creating the stack twice then... Is that intentional?

Reply:
Good point, thanks!